src/app/components/radio/index.html

```diff
@@ -1,13 +1,268 @@
+        </sky-demo-page-content>
+        
+        <sky-demo-page-content sectionHeading="Content">
+        
+            <div class="sky-text-block">
+                <p>Radio buttons may or may not use a preceeding question or statement to provide context for the options. When one is not included, provide the context for the consequences of the choice in the radio button text labels themselves.</p>
+                <p>Icon radio buttons use the <a href="https://fontawesome/v4.7.0/">FontAwesome 4.7</a> icon library. See the <a routerLink="/design/styles/icons">SKY UX icon dictionary</a> for a list of common icons.</p>
+                <p><b>Note:</b> Icon radio buttons must still use a <sky-code>label</sky-code> or <sky-code>labelledby</sky-code> input for accessibility.</p>
+            </div>
+        
+        </sky-demo-page-content>
+        
+        <sky-demo-page-content sectionHeading="Layout">
+        
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Arrange a group of traditional radio buttons in a vertical stack to make it as easy as possible to see all options, scan their labels, and see which option is selected.</p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div class="sky-margin-stacked-content-section">
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-layout-1.png" />
+                    <div class="caption caption-success sky-text-success">Do stack radio buttons vertically.</div>
+                    </div>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-layout-2.png" />
+                    <div class="caption caption-danger sky-text-danger">Don't arrange radio buttons horizontally.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Use standard SKY UX form group spacing to separate radio button groups from the form elements around them.</p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-layout-3.png" />
+                    <div class="caption caption-success sky-text-success">Do visually separate radio groups from the inputs before and after them.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+        
+        </sky-demo-page-content>
+        
+        <sky-demo-page-content sectionHeading="Related information">
+            <h3>Components</h3>
+			<ul>
+				<li><a routerLink="/components/action-button">Action button</a></li>
```

Comment:
Should link to the Form component page as well since it contains the class for separating the radio group from the items before/after it.

Reply:
Good call. Updating.